generated/nidaqmx/_library_interpreter.py

```diff
@@ -1774,7 +1774,7 @@ def create_task(self, session_name):
         error_code = cfunc(
             session_name, ctypes.byref(task))
         check_for_error(error_code)
-        return task
+        return True, task
```

Comment:
Per your description, I think `True` here is "session initialized boolean", which makes sense. Two things:
1. Is there a good way to indicate that here? Maybe creating a variables set to constant `True` that better clarifies what it means.
2. The gRPC interpreter doesn't do this, yet... that seems like a bad mismatch.

Reply:
In `library_interpreter` I have added a private variable called `new_session_initialized` which will be set to `True` and reused where ever necessary. I have also updated the `gRPC_interpreter` to return the session initialized boolean from the response.